### PR TITLE
Add a link to Dockerfile for default image

### PR DESCRIPTION
**Pull request by betatim**
Took me a while to find the configuration for the default image of the chart so added a link to the guide.

### CODE REVIEW

File: doc/source/user-environment.rst [hunk 1]
```diff
@@ -33,6 +33,11 @@ Use an existing Docker image
    match the version installed by the helm chart that you're using. For example,
    ``v0.5`` of the helm chart uses  ``jupyterhub==0.8``.
 
+.. note::
+
+   You can find the configuration for the default Docker image used in this
+   guide `here <https://github.com/jupyterhub/zero-to-jupyterhub-k8s>`_.
```
Comment (choldgraf) [inline doc/source/user-environment.rst]: I'm a bit confused - there's no Dockerfile in the root of the z2jh repository...is that the right link?

Comment (cam72cam) [reply doc/source/user-environment.rst]: Might be looking for https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/images/singleuser-sample/Dockerfile which is broken until https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/439 is merged

**Comment by cam72cam** [comment]
This would have saved me at least a half hour :+1:

File: doc/source/user-environment.rst (hunk 1 above)
Comment (betatim) [reply doc/source/user-environment.rst]: Indeed, pasting links while not paying attention 🚓

Comment (betatim) [reply doc/source/user-environment.rst]: I linked to the directory so people also get the README etc.

Comment (yuvipanda) [reply doc/source/user-environment.rst]: We need to rewrite the README completely, since it's 100% out of date...

**Comment by yuvipanda** [comment]
Thanks for the PR, @betatim!

**Pull request merged**